test/e2e/categories_test.go

```diff
@@ -38,7 +38,7 @@ const (
 	defaultNonExistingAdditionalCategoryValue = "nonExistingCategoryValueCAPX"
 )
 
-var _ = Describe("Nutanix categories", func() {
+var _ = Describe("Nutanix categories", Label("capx-feature-test", "slow", "network"), func() {
```

Comment:
Please add a specific label so we can target the specific categories tests

Reply:
added